app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/pairing/ServiceDiscoveryFragment.kt

```diff
@@ -27,8 +28,9 @@ import kotlinx.android.synthetic.main.fragment_connecting_devices.*
 import java.util.concurrent.TimeUnit
 import javax.inject.Inject
 
-class ServiceDiscoveryFragment : BaseDaggerFragment() {
+class ServiceDiscoveryFragment : BaseFragment() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/a540cb343de5354d91c6cd7978445c3390e2e9c1/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/pairing/ServiceDiscoveryFragment.kt#L31"></span>Class ‘ServiceDiscoveryFragment’ with ‘15’ functions detected. Defined threshold inside classes is set to ‘15’</td>
    </tr>
  </tbody>
</table>